frontend/kesaseteli/employer/src/components/application/steps/step2/accordions/EmploymentAccordion.tsx

```diff
@@ -29,6 +33,27 @@ type Props = {
 
 const EmploymentAccordion: React.FC<Props> = ({ index }: Props) => {
   const { t } = useTranslation();
+  const { getValues } = useFormContext<Application>();
+  const { applicationQuery, fetchEmployment } = useApplicationApi();
+  const [isFetchEmployeeDataEnabled, setIsFetchEmployeeDataEnabled] =
+    useState<boolean>(false);
+
+  useEffect(() => {
+    enableFetchEmployeeDataButton();
+  }, []);
+
+  const enableFetchEmployeeDataButton = () => {
+    const formDataVoucher = getValues().summer_vouchers[index];
+    setIsFetchEmployeeDataEnabled(
+      formDataVoucher.employee_name.length > 0 &&
+        formDataVoucher.summer_voucher_serial_number.length > 0
+    );
+  };
```

Comment:
They should be other way around, so I would know what is called.

Reply:
i dont understand what you mean

Reply:
yep, that I meant